addon/mixins/routes/application.js

```diff
@@ -31,6 +31,13 @@ export default Ember.Mixin.create({
         outlet: outlet,
         parentView: parentView
       });
+
+      // Check if another modal exists then give it focus
+      Ember.run.later(function() {
+        var remainingViewElement = Ember.$('[role="dialog"]:last-child').closest('.ember-view');
```

Comment:
Do you need to give the focus to the overlay or can you just give it to the modal dialog (e.g. not use `closest()`). The reason I ask is because `.closest('.ember-view')` seems a bit brittle.